operations/mimir-mixin/scripts/lint-playbooks.sh

```diff
@@ -16,11 +18,29 @@ elif [ -z "$ALERTS" ]; then
 fi
 
 # Check if each alert is referenced in the playbooks.
-STATUS=0
-
 for ALERT in $ALERTS; do
-  if ! grep -q "${ALERT}$" "${SCRIPT_DIR}/../docs/playbooks.md"; then
-    echo "Missing playbook for: $ALERT"
+  if ! grep -q "# ${ALERT}$" "${SCRIPT_DIR}/../docs/playbooks.md"; then
+    echo "Missing playbook for alert: $ALERT"
+    STATUS=1
+  fi
+done
+
+# List all global error IDs.
+if ! ERROR_IDS=$(${SED_BIN} --quiet -E 's/^.*ErrID\s+=\s+"([^"]+)"$/\1/p' "${SCRIPT_DIR}/../../../pkg/util/globalerrors/errors.go"); then
+  echo "Unable to list error IDs. Got output:"
+  echo "$ERROR_IDS"
+  exit 1
+elif [ -z "$ERROR_IDS" ]; then
+  echo "No error IDs found. Something went wrong with the listing."
```

Comment:
Is this always plural? If not, please use "No error ID found. ..."

Reply:
We always expect to find many error IDs. If none was found, then there's a bug with this script.